ci-operator/jobs/openshift-knative/serving/openshift-knative-serving-release-next-postsubmits.yaml

```diff
@@ -10,10 +10,10 @@ postsubmits:
       skip_cloning: true
     labels:
       ci-operator.openshift.io/is-promotion: "true"
-      ci-operator.openshift.io/variant: "412"
```

Comment:
Was this, and similar changes (12 -> 10), intended?

Reply:
I configured `ci-operator/config/*` but other files such as `ci-operator/jobs/*` are auto updated via `make jobs` (the script is owned by this repo's not ours).

So, this file is generated by `make jobs` automatically and it is not my intention but the automated script's intention.

Reply:
Ah yep, I see. Still confusing :)